next.config.js

```diff
@@ -34,5 +31,8 @@ module.exports = {
   },
   eslint: {
     ignoreDuringBuilds: true
+  },
+  images: {
+    unoptimized: true
   }
```

Comment:
The configuration we had [was a work around for `next export`](https://github.com/vercel/next.js/issues/27231#issuecomment-888414710), which no longer works with next 13

So I removed it which results in us using the default image loading mechanism. See
    
    https://nextjs.org/docs/messages/next-image-missing-loader

But that is not compatible with next export either, so I added the `unoptimized: true` configuration in response to the cli error message:

```
> next export

info  - Loaded env from /home/richard/projects/cert-manager/website/.env
info  - using build directory: /home/richard/projects/cert-manager/website/.next
info  - Copying "static build" directory
info  - No "exportPathMap" found in "/home/richard/projects/cert-manager/website/next.config.js". Generating map from "./pages"
Error: Image Optimization using Next.js' default loader is not compatible with `next export`.
  Possible solutions:
    - Use `next start` to run a server, which includes the Image Optimization API.
    - Configure `images.unoptimized = true` in `next.config.js` to disable the Image Optimization API.
  Read more: https://nextjs.org/docs/messages/export-image-api
    at /home/richard/projects/cert-manager/website/node_modules/next/dist/export/index.js:153:23
    at async Span.traceAsyncFn (/home/richard/projects/cert-manager/website/node_modules/next/dist/trace/trace.js:79:20)


```